cmd/object-handlers.go

```diff
@@ -53,6 +53,7 @@ import (
 	"github.com/minio/minio/internal/handlers"
 	"github.com/minio/minio/internal/hash"
 	xhttp "github.com/minio/minio/internal/http"
+	httplogger "github.com/minio/minio/internal/http/logger"
```

Comment:
Why are you aliasing it? It is already called `package httplogger` - it will work just fine without the alias.

But decide on what it should be called and name the directory the same. `loghttp` is a bit shorter and fine.

Reply:
> Why are you aliasing it? It is already called package httplogger - it will work just fine without the alias.

yes right

> But decide on what it should be called and name the directory the same. loghttp is a bit shorter and fine

loghttp is probably redunant because we are in internal/http already, what I can do is rename this to logger but keeps httplogger alias

This is my two cents anyway, especially this is not a public API

Reply:
Yeah, but directories and package names should match.

You are also welcome to move it to our own http package.

`http.RequestRecorder` makes perfect sense.

Reply:
> http.RequestRecorder makes perfect sense.

oh right, will do that